src/Controllers/ExampleController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Controllers;
+
+
+use App\Models\ExampleModel;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Slim\Http\Response;
+
+class ExampleController
+{
+    protected ExampleModel $model;
+
+    public function __construct(ExampleModel $model)
+    {
+        $this->model = $model;
+    }
+
+    public function __invoke(RequestInterface $request, Response $response): ResponseInterface
+    {
+        return $response->withJson(['data' => 'fake data']);
```

Comment:
i would suggest here as well to write the code that pulls data from the model and use that data to provide to the response

another suggestion would be to reinforce RESTful practices by putting the response data in a standard-ish format:
```php
$tweets = $this->tweetModel->getAllTweets(); // using TweetModel for a more realistic (imo) example

$responseBody = [
   'message' => 'Tweets successfully retrieved from db.',
   'status' => 200,
   'data' => $tweets
];
return $response->withJson($responseBody);
```

in my experience providing the full example upfront saves later having to clarify things individually, such as how to combine a controller and a model.

let me know what you think though